linalg/AmgXSolver.hpp

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
```

Comment:
This hpp and the corresponding cpp file would be the first to begin with a capital letter. Maybe they should be renamed in lower-case.